include/tune_quda.h

```diff
@@ -243,7 +243,33 @@ namespace quda {
     virtual void apply(const qudaStream_t &stream) = 0;
     virtual void preTune() { }
     virtual void postTune() { }
-    virtual int tuningIter() const { return 1; }
+
+    /**
+     * @brief Parameter to control the number of iteration used in the 2nd phase of tuning, i.e. for the candidates.
+     *
+     * @return int minimum number of iterations
```

Comment:
The `return` statement shouldn't include the type being returned (same applies for the next few functions in this file).